src/bundling/Bundler.js

```diff
@@ -166,8 +166,17 @@ class Bundler {
      * a message row, date divider, or bundle row.
      */
     _calculateSortedTableRows(messageNodes, bundlesByLabel) {
+
+        chrome.storage.sync.get(['bundleDates'], ({ bundleDates = true }) => {
+            this.bundleDates = bundleDates;
+        });
+        
```

Comment:
The general existing pattern in inboxy is that when you change your settings, the new settings aren't observed until you refresh the page. 

I'm not sure whether that's ideal, but just to stay consistent, I think these lines could be moved into the constructor, or possibly somehow pass the setting as a param to the constructor.

Reply:
Great feedback, thanks. The last thing you want is inconsistent behaviour for different settings, so hoisting to the constructor SGTM.